cilium/cmd/endpoint_config.go

```diff
@@ -20,6 +20,7 @@ import (
 
 	"github.com/cilium/cilium/pkg/command"
 	"github.com/cilium/cilium/pkg/endpoint"
+	endpointid "github.com/cilium/cilium/pkg/endpoint/id"
```

Comment:
Merge this commit with the previous commit in order not to break git bisecting as the previous commit alone will not compile.